primeqa/ir/dense/colbert_top/colbert/utils/parser.py

```diff
@@ -28,7 +28,9 @@ def add_model_parameters(self):
         self.add_argument('--doc_maxlen', dest='doc_maxlen', default=180, type=int)
 
         # Filtering-related Arguments
-        self.add_argument('--mask-punctuation', dest='mask_punctuation', default=False, action='store_true')
+        self.add_argument('---mask-punctuation', dest='mask_punctuation', action='store_true')
```

Comment:
Why is this argument needed if it's the default behavior? If you keep both, have you tested what happens? 
```suggestion
        self.add_argument('--mask-punctuation', dest='mask_punctuation', action='store_true')
```